Add Meter Provider documentation (#469)
aws-otel/aws-otel.github.io: src/docs/getting-started/lambda/lambda-python.mdx

 which exports telemetry data to AWS X-Ray. To customize the Collector config,
 see the [main Lambda section for custom configuration instructions](/docs/getting-started/lambda#custom-configuration-for-the-adot-collector-on-lambda)

> The ADOT Python Lambda Layer can also be configured to set a default Meter Provider with readers if the OTEL_METRICS_EXPORTER environment variable is set to the values noted here:

Reviewer: ```suggestion
The ADOT Python Lambda Layer can also be configured to set a Meter Provider with readers if the `OTEL_METRICS_EXPORTER` environment variable is set to the values noted here:
```

> https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection.  The OTEL_EXPORTER_OTLP_PROTOCOL environment variable should also be set

Reviewer: ```suggestion
https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection.  The `OTEL_EXPORTER_OTLP_PROTOCOL` environment variable should also be set
```

> The ADOT Python Lambda Layer can also be configured to set a Meter Provider with readers if the `OTEL_METRICS_EXPORTER` environment variable is set to the values noted here:
> https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection.  The `OTEL_EXPORTER_OTLP_PROTOCOL` environment variable should also be set
> in correlation to the metrics exporter with these supported values: https://opentelemetry.io/docs/reference/specification/protocol/exporter/#specify-protocol.

Reviewer 2: ```suggestion
The ADOT Python Lambda Layer can also be configured to set a Meter Provider with an appropriate reader and exporter if the `OTEL_METRICS_EXPORTER` environment variable is set to one of the values noted [here](https://opentelemetry.io/docs/reference/specification/sdk-environment-variables/#exporter-selection).  The `OTEL_EXPORTER_OTLP_PROTOCOL` environment variable should also be set
in correlation to the metrics exporter with [these supported values](https://opentelemetry.io/docs/reference/specification/protocol/exporter/#specify-protocol).
```

We may also want to change [this setting](https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel-instrument#L99) in the Python Lambda wrapper script as it defaults the `OTEL_METRICS_EXPORTER` value to `none` rather than `otlp` as that document indicates.

Author: Yeah I noticed that when I was researching.  I'm not sure what the desired default behavior is, but should I open up another PR for that upstream?
